resources/centos7/after_install.sh

```diff
@@ -34,253 +37,51 @@ case "$1" in
             mkdir -p "$PLUGIN_HOME"
         fi
 
-        if [ ! -d "$PID_HOME" ]; then
-          mkdir -p "$PID_HOME"
```

Comment:
This makes me nervous. This came about as the result of a bug where we were not managing PIDs correctly. This might be gone moving to the services, but I feel its necessary to confirm that:

1. Start any beer-garden service
2. Stop the service
3. Ensure that the python process is no longer running

The reason we needed to do this in the first place was because if this directory was not created, then the pid file would not get created and our service scripts could not actually stop the services. So as long as we can do that, I'm good.